### PR TITLE
[onert_run] Support different random input on each rerun

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates random generator to support different random input on each rerun.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #13008

### CODE REVIEW

**Comment by hseok-oh** [comment]
#### With `--fixed_input`

```
$ MINMAX_DUMP=1 ./Product/out/bin/onert_run --fixed_input -r 3 /home/nfs/model/mobilenet/mobilenet_v1_1.0_224.circle

$ h5dump minmax.h5

      GROUP "run_0" {
         GROUP "model_0" {
            GROUP "subg_0" {
               DATASET "input_0" {
                  DATATYPE  H5T_IEEE_F32BE
                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
                  DATA {
                  (0): -9.82522, 8.8199
                  }
               }

      GROUP "run_1" {
         GROUP "model_0" {
            GROUP "subg_0" {
               DATASET "input_0" {
                  DATATYPE  H5T_IEEE_F32BE
                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
                  DATA {
                  (0): -9.82522, 8.8199
                  }
               }

      GROUP "run_2" {
         GROUP "model_0" {
            GROUP "subg_0" {
               DATASET "input_0" {
                  DATATYPE  H5T_IEEE_F32BE
                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
                  DATA {
                  (0): -9.82522, 8.8199
                  }
               }
```

#### Without `--fixed_input`

```
$ MINMAX_DUMP=1 ./Product/out/bin/onert_run --fixed_input -r 3 /home/nfs/model/mobilenet/mobilenet_v1_1.0_224.circle

$ h5dump minmax.h5

      GROUP "run_0" {
         GROUP "model_0" {
            GROUP "subg_0" {
               DATASET "input_0" {
                  DATATYPE  H5T_IEEE_F32BE
                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
                  DATA {
                  (0): -9.82522, 8.8199
                  }
               }

      GROUP "run_1" {
         GROUP "model_0" {
            GROUP "subg_0" {
               DATASET "input_0" {
                  DATATYPE  H5T_IEEE_F32BE
                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
                  DATA {
                  (0): -8.82572, 11.1739
                  }
               }

      GROUP "run_2" {
         GROUP "model_0" {
            GROUP "subg_0" {
               DATASET "input_0" {
                  DATATYPE  H5T_IEEE_F32BE
                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
                  DATA {
                  (0): -8.77365, 9.77205
                  }
               }
```

File: tests/tools/onert_run/src/args.cc [hunk 1]
```diff
@@ -272,6 +272,7 @@ void Args::Initialize(void)
         "If not given, the model's output sizes are used\n"
         "e.g. '[0, 40, 2, 80]' to set 0th tensor to 40 and 2nd tensor to 80.\n")
     ("num_runs,r", po::value<int>()->default_value(1)->notifier([&](const auto &v) { _num_runs = v; }), "The number of runs")
+    ("fixed_input", "Use same random input data on each run (avaliable on random input)")
```
Comment (jyoungyun) [inline tests/tools/onert_run/src/args.cc:275]: (just for curiosity) Is there a reason why you don't use the same format as others?

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM

File: tests/tools/onert_run/src/args.cc (hunk 1 above)
Comment (hseok-oh) [reply tests/tools/onert_run/src/args.cc:275]: If we use `po::value<bool>()->default_value(false)->notifier([&](const auto &v) ...`, then we need to set `--fixed_input true` (not `--fixed_input`) to enable this feature.

**Pull request merged**